Samples/PowerShell/NTA.DownloadRouterConfigFromNCM.ps1

```diff
@@ -0,0 +1,54 @@
+# This sample script demonstrates how to download configuration for Router
+# from NCM. You can use it to verify Netflow related configuration on the Router.
+
+
+# Connect to SWIS
+$hostname = "swis-machine-hostname"                     # Update to match your configuration
+$username = "admin"                                     # Update to match your configuration
+$password = New-Object System.Security.SecureString     # Update to match your configuration
+$cred = New-Object -typename System.Management.Automation.PSCredential -argumentlist $username, $password
+$swis = Connect-Swis -host $hostname -cred $cred
+
+# Get Orion Node, Interface, Netflow Source information and Router identifiers for concrete Netflow Source
+# You can use these alternative conditions
+# If you know NetflowSourceID: WHERE S.NetflowSourceID = $netflowSourceId
+# If you know NodeName: WHERE N.NodeName = '$nodeName'
+# If you know InterfaceName: WHERE I.InterfaceName = '$interfaceName'
+$nodeName = "my.netflownode.corp"       # Update to match your configuration
+$interfaceName = "GigabitEthernet0/1"   # Update to match your configuration
+
+$query= "
+SELECT s.NetflowSourceID, s.NodeID, s.InterfaceID, s.Enabled, s.LastTimeFlow, s.LastTime, s.EngineID, 
+    s.Node.NodeName,
+    s.Interface.Name as InterfaceName, s.Interface.Index as RouterIndex
+FROM Orion.Netflow.Source s
+WHERE s.Node.NodeName = '$nodeName' AND s.Interface.InterfaceName = '$interfaceName'
+"
+$netflowSourceInfo = Get-SwisData $swis $query
+if(!$netflowSourceInfo) 
+{
+    Write-Error "Netflow Source information not found" -ErrorAction Stop
+}
```

Comment:
I would update these to use the "One True Brace Style" variant of K&R for readability.

Reply:
Fixed